### ISSUE TITLE
Sort Options Should Have Focus State

### ISSUE THREAD

**Issue by rosiegrant** [yext/answers-search-ui#1036]
Per WCAG best practice, the sort options should have a focus state. Would make sense to match what we do for filter options and facets, https://github.com/yext/answers/blob/master/src/ui/sass/modules/_FilterOptions.scss#L51

```
.yxt-SortOptions-radio_SortOptions:focus {
   var(--yxt-border-hover);
}
```
